### PR TITLE
[TBT-222] add small text for trials 14 days

### PR DESCRIPTION
**Pull request by nikolaTrichkovski**

### CODE REVIEW

File: app/templates/components/billing/payment.hbs [hunk 1]
```diff
@@ -115,6 +115,9 @@
             <p>{{stripeError.message}}</p>
           {{/if}}
         </StripeCard>
+        <span class="trial-no-charge-text">
+          Your credit card will not be charged for 14 days. Enjoy your trial!
+        </span>
```
Comment (DominikAlberski) [inline app/templates/components/billing/payment.hbs:120]: This message should be shown for users starting a trial, but I don't see any condition for that.

Comment (nikolatravis) [reply app/templates/components/billing/payment.hbs]: @DominikAlberski thanks, I added it.

Comment (nikolaTrichkovski) [reply app/templates/components/billing/payment.hbs]: @DominikAlberski thanks, added.

**Review by DominikAlberski — APPROVED** [review]

**Review by murtaza-swati — COMMENTED** [review]
@nikolaTrichkovski  I am not sure if this message "no charges for 14 days" with be also on page /account_activation
I think the page for that is components/billing/first-plan.hbs

**Comment by nikolaTrichkovski** [comment]
> @nikolaTrichkovski I am not sure if this message "no charges for 14 days" with be also on page /account_activation I think the page for that is components/billing/first-plan.hbs

@murtaza-swati if you mean if it is in the modal while on page /account_activation it is.
Screenshot 2024-11-15 at 09 53 59

File: app/templates/components/billing/select-plan.hbs [hunk 2]
```diff
@@ -33,7 +33,7 @@
             {{#if (or (and plan.isAnnual this.showAnnual) (and (not plan.isAnnual) (not this.showAnnual)))}}
               <div
                 data-test-plan-box='true'
-                class='billing-plans__box-v2 pointer {{if (eq plan.id this.selectedPlan.id) 'highlight-plan'}}'
+                class='billing-plans__box-v2 pointer {{if (or (eq plan.id this.selectedPlan.id) (eq plan.name "Fixed Price: 5 Jobs")) "highlight-plan"}}'
```
Comment (murtaza-swati) [inline app/templates/components/billing/select-plan.hbs:36]: @nikolaTrichkovski 
this naming of plans is something that keeps changing.
now it is "Fixed Price: 5 Jobs", previously it was something and in future we don't know what could it be next.
I believe there should be a better way to handle it

**Review by murtaza-swati — CHANGES_REQUESTED** [review]

**Comment by murtaza-swati** [comment]
> > @nikolaTrichkovski I am not sure if this message "no charges for 14 days" with be also on page /account_activation I think the page for that is components/billing/first-plan.hbs
> 
> @murtaza-swati if you mean if it is in the modal while on page /account_activation it is. Screenshot 2024-11-15 at 09 53 59

@nikolaTrichkovski I mean this:
Screenshot 2024-11-15 at 1 56 18 PM

**Comment by nikolaTrichkovski** [comment]
> > > @nikolaTrichkovski I am not sure if this message "no charges for 14 days" with be also on page /account_activation I think the page for that is components/billing/first-plan.hbs
> > 
> > 
> > @murtaza-swati if you mean if it is in the modal while on page /account_activation it is. Screenshot 2024-11-15 at 09 53 59
> 
> @nikolaTrichkovski I mean this: Screenshot 2024-11-15 at 1 56 18 PM

@murtaza-swati Is it for the text: Your credit card won't be charged for 14 days. Enjoy your trial!"?
In that case it is not shown there, I will add it of course

**Pull request merged**